djstripe/models/core.py

```diff
@@ -582,8 +582,10 @@ def __str__(self):
 
         if price_qs.count() > 1:
             return f"{self.name} ({price_qs.count()} prices)"
-        else:
+        elif price_qs.count() == 1:
```

Comment:
You're calling price_qs.count() twice, that makes an unnecessary sql call.

Let's just do this:

```py
if price_qs.count() == 1:
    return f"{self.name} ({price_qs[0].human_readable_price})"
else:
    return f"{self.name} ({price_qs.count()} prices)"
```